### DISCUSSION TITLE
Utility Meter helper source

### DISCUSSION THREAD

**Discussion by NateLogan** [home-assistant/frontend#13273] [Configuration] [answered]
If the Utility Meter is configurred through UI helper, there appears to be no option to change that configuration.

**Comment by frenck** [comment, marked as answer]
![CleanShot 2022-07-25 at 13 43 37](https://user-images.githubusercontent.com/195327/180770230-b7614cac-cf43-4424-bdec-d3d7d7b7aa27.png)
